source/MQTTFileDownloader.c

```diff
@@ -81,13 +97,37 @@ static MQTTFileDownloaderStatus_t handleCborMessage( uint8_t * decodedData,
  * @param[in] messageLength Length of the MQTT message received.
  * @param[out] decodedData Buffer to place the decoded data in.
  * @param[out] decodedDataLength Length of decoded data.
+ * @param[out] fileId ID of the file to which the data block belongs.
+ * @param[out] blockId ID of the received block.
+ * @param[out] blockSize Size of the receive block in bytes.
  *
  * @return uint8_t returns appropriate MQTT File Downloader Status.
  */
```

Comment:
It feels odd to me that the parameters in the comment aren't in order with the actual function deceleration?

Reply:
Good catch. Fixed with https://github.com/aws/aws-iot-core-mqtt-file-streams-embedded-c/pull/23/commits/f1f6bdba40afd6af472fc0412d5ea99a555623f7